en/dev_log/ulog_file_format.md

```diff
@@ -23,14 +29,12 @@ They all correspond to the types in C:
 |double             |  8            |
 |bool, char         |  1            |
 
-Additionally all can be used as an array, eg. `float[5]`.
-In general all strings (`char[length]`) do not contain a `'\0'` at the end.
+Additionally the types can be used as an array : eg. `float[5]`. In case of logging of strings : (`char[length]`), it usually does not contain the termination NULL character `'\0'` at the end.
 String comparisons are case sensitive.
 
+## ULog structure
 
-## File structure
-
-The file consists of three sections:
+ULog consists of the following three sections:
```

Comment:
You're talking about a file here, so you wouldn't normally rename to "ulog", and if you do, you'd say "the ULog file format".

SO this would reasonably change to
```
The ULog file format consists of the following three sections:
```
or 
```
The file consists of the following three sections:
```

Reply:
I thought ULog itself is a file format, therefore there is no need to state it again (even the document's name is `ulog_file_format.md`)

Reply:
YOu could be right - not sure it is defined as such though. But would you say "Zip consists of" or "Mission plan consists of"?

Reply:
Hmm yea that wouldn't make much sense 🤔